scrapy/utils/asyncio.py

```diff
@@ -0,0 +1,26 @@
+#coding: utf-8
+
+
+def install_asyncio_reactor():
+    """ Tries to install AsyncioSelectorReactor
```

Comment:
This currently just tries to install the reactor, it can also return if the reactor is the asyncio one and/or set some var. See also is_asyncio_supported below. 

Note that checking the reactor directly, like is_asyncio_supported does, it better than checking a var set or returned from install_asyncio_reactor, as the reactor could be installed outside it.